print_list.rb

```diff
@@ -0,0 +1,27 @@
+puts "Bienvenido al programa, Imprimir lista"
+puts "Ingresa el numero de personas"
+lenghtList = gets.chomp.to_i
+personas = Array.new
```

Comment:
check this [ruby-style-guide#literal-array-hash](https://github.com/rubocop/ruby-style-guide#literal-array-hash)
```suggestion
personas = []
```